packages/zowe-explorer/src/extension.ts

```diff
@@ -39,6 +42,25 @@ export async function activate(context: vscode.ExtensionContext): Promise<ZoweEx
     await ProfilesUtils.handleV1MigrationStatus();
     await Profiles.createInstance(ZoweLogger.imperativeLogger);
 
+    const paginationEnabled = SettingsConfig.getDirectValue<boolean>("zowe.jobs.settings.pagination");
+    if(paginationEnabled) {
+        context.subscriptions.push(
+            vscode.commands.registerCommand("zowe.jobs.loadMoreRecords", async () => {
+                const editor = vscode.window.activeTextEditor;
+                if (!editor) {
+                    vscode.window.showErrorMessage("No active editor found.");
+                    return;
+                }
+
+                const document = editor.document;
+                const uri = document.uri;
+                if (uri.scheme == ZoweScheme.Jobs) {
+                    await JobFSProvider.instance.fetchSpoolAtUri(uri);
+                }
+            }),
+            vscode.languages.registerCodeLensProvider({ scheme: ZoweScheme.Jobs }, new PaginationCodeLens("zowe.jobs.loadMoreRecords"))
+        );
+    }
```

Comment:
Would it make sense to move the registration of this command inside the `JobInit.initJobsProvider` method?
https://github.com/zowe/zowe-explorer-vscode/blob/c6d656daf71575819658ce2d1ab392a7017cd7b2/packages/zowe-explorer/src/trees/job/JobInit.ts#L42

Generally we try to keep `extension.ts` as small as possible and register commands elsewhere 🙂